.github/workflows/main.yml

```diff
@@ -51,44 +60,44 @@ jobs:
           pip install -r requirements_docs.txt
           Copy-Item -Path "C:\actions-runner\opengl32.dll" -Destination "testenv\Lib\site-packages\vtkmodules" -Force
- 
+
         #if: startsWith(github.event.ref, 'refs/heads/main') != true
-      
-      - name: 'Create Documentations'      
+
+      - name: 'Create Documentations'
         run: |
           testenv\Scripts\Activate.ps1
           sphinx-build -j auto -b html -a doc/source doc/_build/html 
-          
-      - name: ' Deploy on aedtdocs.pyansys.com'
-        if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags')
-        working-directory: doc/_build/html
-        run: |
-          git init
-          git checkout -b master
-          git config --local user.name "maxcapodi78"
-          git config --local user.email "massimo.capodiferro@ansys.com"
-          New-Item -ItemType file .nojekyll
-          New-Item -ItemType file CNAME
-          Add-Content CNAME "aedtdocs.pyansys.com"
-          git add .
-          git commit -m "Documentation generated by PyAedt Build"
-          git remote add origin https://${{ secrets.PYAEDT_DOCS }}@github.com/pyansys/PyAEDT-docs
-          git push -u origin master --force
+
+      # - name: ' Deploy on aedtdocs.pyansys.com'
+      #   if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags')
+      #   working-directory: doc/_build/html
+      #   run: |
+      #     git init
+      #     git checkout -b master
+      #     git config --local user.name "maxcapodi78"
+      #     git config --local user.email "massimo.capodiferro@ansys.com"
+      #     New-Item -ItemType file .nojekyll
+      #     New-Item -ItemType file CNAME
+      #     Add-Content CNAME "aedtdocs.pyansys.com"
+      #     git add .
+      #     git commit -m "Documentation generated by PyAedt Build"
+      #     git remote add origin https://${{ secrets.PYAEDT_DOCS }}@github.com/pyansys/PyAEDT-docs
+      #     git push -u origin master --force
 
 
-      #- name: Upload Documentation
-      #  uses: actions/upload-artifact@v2.2.3
-      #  with:
-      #    name: Documentation
-      #    path: doc/_build/html
-      #    retention-days: 7
+      - name: Upload Documentation
+        uses: actions/upload-artifact@v2.2.3
+        with:
+          name: Documentation
+          path: doc/_build/html
+          retention-days: 7
 
-      #- name: Deploy
-      #  if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags')
-      #  uses: JamesIves/github-pages-deploy-action@4.1.4
-      #  with:
-      #  repository-name: pyansys/pyaedt-docs
-      #   branch: master
-      #    folder: doc/_build/html/
-      #    token: ${{ secrets.PYANSYS_CI_BOT_TOKEN }}
-      #    clean: true
+      - name: Deploy
+        if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags')
+        uses: JamesIves/github-pages-deploy-action@4.1.4
+        with:
+          repository-name: pyansys/pyaedt-docs
+          branch: gh-pages
```

Comment:
Note that changing the branch we push to will mean we also have to change the "pages" branch used by gh-pages in https://github.com/pyansys/PyAEDT-docs/settings/pages